src/main/resources/assets/dragonsurvival/lang/en_us.json

```diff
@@ -1339,21 +1339,21 @@
   "ds.advancements.transplant_chorus_fruit.title": "Favorite Treat Everywhere",
   "ds.advancements.transplant_chorus_fruit.description": "Transplant a tasty purple flower back home on the Overworld.",
   "ds.advancements.meat_eater.title": "Meat Eater",
-  "ds.advancements.meat_eater.description": "Eat all unique food that is marked as forest dragon food from Dragon Survival. JEI is your friend!",
+  "ds.advancements.meat_eater.description": "Eat all unique forest dragon food from Dragon Survival.",
   "ds.advancements.master_all_forest_passives.title": "Photosynthesis",
   "ds.advancements.master_all_forest_passives.description": "Achieve total mastery of your forest dragon passive skills.",
   "ds.advancements.be_sea_dragon.title": "Heterochromia Iridum",
   "ds.advancements.be_sea_dragon.description": "Become a sea dragon by any means necessary.",
   "ds.advancements.be_old_sea_dragon.title": "True Ancient Guardian",
   "ds.advancements.be_old_sea_dragon.description": "Gain level 60 growth as a sea dragon.",
   "ds.advancements.loot_shipwreck.title": "Ocean Reclaimer",
-  "ds.advancements.loot_shipwreck.description": "Claim salvage from a 3 ship chest that was lost at sea.",
+  "ds.advancements.loot_shipwreck.description": "Claim salvage from all the different chests in a shipwreck.",
```

Comment:
It is better to specify that they are exactly 3 types, because for the advancements you need exactly 3 loot tables.

Reply:
maybe "all three types of chests in a shipwreck"?